contracts/apps/disputable/IAgreement.sol

```diff
@@ -57,18 +51,6 @@ contract IAgreement {
 
     function disputeAction(uint256 _actionId, bool _finishedSubmittingEvidence) external;
 
-    function getSigner(address _signer) external view returns (uint256 lastSettingIdSigned, bool mustSign);
-
-    function getCurrentSettingId() external view returns (uint256);
-
-    function getSetting(uint256 _settingId) external view
-        returns (
-            IArbitrator arbitrator,
-            IAragonAppFeesCashier aragonAppFeesCashier,
-            string title,
-            bytes content
-        );
-
     function getDisputableInfo(address _disputable) external view returns (bool activated, uint256 currentCollateralRequirementId);
 
     function getCollateralRequirement(address _disputable, uint256 _collateralId) external view
```

Comment:
Similarly, I'm wondering if `getDisputableInfo()` and `getCollateralRequirement()` are worth keeping.